edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.CloudProxy/ServiceProxy.cs

```diff
@@ -39,7 +39,18 @@ public async Task<Option<ServiceIdentity>> GetServiceIdentity(string deviceId)
             Option<ScopeResult> scopeResult = Option.None<ScopeResult>();
             try
             {
-                ScopeResult res = await this.securityScopesApiClientProvider.CreateDeviceScopeClient().GetIdentityAsync(deviceId, null);
+                IDeviceScopeApiClient client;
+
+                if (this.nestedEdgeEnabled)
+                {
+                    client = this.securityScopesApiClientProvider.CreateNestedDeviceScopeClient(Option.None<string>());
```

Comment:
just curious, I can't find any usage of this method with non-empty value.  In what scenario we will pass continuation link to this method?

Reply:
hmm....I think this was an oversight from an earlier refactor.  A previous implementation of the nested logic in ServiceProxy only had one creation method for the HTTP client, so it took a continuation token.  Then later the nested portion was moved to CreateOnBehalfOf, so nobody calls this with a continuation token anymore.  I can remove it.

---
In reply to: [458480574](https://github.com/Azure/iotedge/pull/3248#discussion_r458480574) [](ancestors = 458480574)